lldb/include/lldb/Target/RegisterFlags.h

```diff
@@ -17,11 +17,17 @@ class RegisterFlags {
 public:
   class Field {
   public:
+    /// Where start is the least significant bit and end is the most
+    /// significant bit. The start bit must be <= the end bit.
     Field(std::string name, unsigned start, unsigned end)
         : m_name(std::move(name)), m_start(start), m_end(end) {
       assert(m_start <= m_end && "Start bit must be <= end bit.");
     }
 
+    /// Construct a single bit field.
+    Field(std::string name, unsigned bit)
+        : m_name(std::move(name)), m_start(bit), m_end(bit) {}
```

Comment:
Can you either add more to the doxygen comment or rename the `bit` field? Perhaps something like `bit_position` or something? It feels a little vague to me at the moment.

Reply:
Will do.

Reply:
Also it does read like "a single bitfield" instead of "a single bit field" doesn't it, I'll make it more obvious.

Reply:
Done as suggested.